Extension/package.nls.json

```diff
@@ -36,6 +36,7 @@
     "c_cpp.command.selectSshTarget.title": "Select SSH target",
     "c_cpp.command.activeSshTarget.title": "Get the active SSH target",
     "c_cpp.command.refreshCppSshTargetsView.title": "Refresh",
+    "c_cpp.command.sshTerminal.title": "Connect to this target in a new terminal",
```

Comment:
Should "target" be "SSH target"? I'm concerned the translators may not know how to translate it consistently with the other references to "SSH target" in the other strings. Maybe a "comment" to translators could be added instead.